components/Preferences.tsx

```diff
@@ -25,7 +25,7 @@ import { useForm } from "react-hook-form";
 import StatusIcon from "./core/StatusIcon";
 import ToastWrapper from "./core/ToastWrapper";
 
-function Preferences() {
+function Preferences(props: { isUnsubscribing: boolean }) {
```

Comment:
Interesting you have to pass this variable here, rather than being able to get it from `useSubscription()`

Reply:
`isUnsubscribing` is true when running `unsubscribe` and since the hooks do not come from a universal context (I.E we don't force people to wrap their `App.tsx` in a `<Web3InboxContext>` like Wagmi, we don't know if someone is unsubsribing.

This was a design decision